parsl/providers/kubernetes/kube.py

```diff
@@ -1,86 +1,107 @@
 import logging
 import time
+from parsl.providers.kubernetes.template import template_string
+
 logger = logging.getLogger(__name__)
 
+from parsl.providers.error import *
 from parsl.providers.provider_base import ExecutionProvider
-from parsl.providers.kubernetes.template import template_string
-from parsl.providers.error import OptionalModuleMissing
-
+from parsl.utils import RepresentationMixin
 
 try:
-    from kubernetes import client
-    from kubernetes import config as kube_config
+    from kubernetes import client, config
+    config.load_kube_config()
```

Comment:
Do we need to load the kube config here ? This try catch will set _kube_enabled to false for 2 different reasons and the user won't know if there was just a problem with the load_kube_config.

Reply:
Moved `config.load_kube_config()` to init function in KubernetesProvider